sdks/nodejs/README.md

```diff
@@ -0,0 +1,24 @@
+### Generate the client library
+
+```
+npm install -g grpc-tools

```

Comment:
If I read this right - is this a developer tooling readme?

Reply:
Yes, I just threw all my developer notes in one file so not sure if all required or if we should split documentation.
The actual usage instructions are not included, although the example shows usage it could perhaps benefit from some basic instructions, such as run `npm install` followed by `npm run`

Reply:
This is the first time I used gRPC and feel I had to resort to a couple of hacks to get something that worked so would appreciate help on this.
Potential hacks:
1. Manually cloning the googleapis repo and including in path
2. Manually removing the lines from `_pb.js` files that import/require `annotations_pb.js` - I couldn't figure out how to generate these automatically and have in the correct location. Could probably have generated from the googleapis project and copied

What's weird is the import is not used in the generated js, but I cannot remove from the .proto as it is required there. This could also be a bug with js code generation

Reply:
Yeeep - yep, we don't want to be doing that :) (manually removing lines from generated code) - but also want to make sure this can continue to work as we upgrade grpc versions, etc.

grpc generation does generate both client and server side code -- so it's not surprising you are getting both.

You can see in our build image, we have scripting to pull in the googleapis information from the right place, and incorporate it where it needs to be:
https://github.com/GoogleCloudPlatform/agones/blob/master/build/build-image/gen-grpc-go.sh
https://github.com/GoogleCloudPlatform/agones/blob/master/build/build-image/gen-grpc-cpp.sh

Does that help?

Reply:
Have now removed instructions and manual steps, and added `gen-grpc-nodejs.sh` as suggested. This changed the generated code as the version of protoc changed but have successfully tested against the local SDK server.